include/dxc/DxilContainer/DxilContainer.h

```diff
@@ -581,6 +581,7 @@ enum class SerializeDxilFlags : uint32_t {
   StripReflectionFromDxilPart = 1 << 3, // Strip Reflection info from DXIL part.
   IncludeReflectionPart       = 1 << 4, // Include reflection in STAT part.
   StripRootSignature          = 1 << 5, // Strip Root Signature from main shader container.
+  SetRootSignature            = 1 << 6, // Set Root Signature in the container from provided source.
```

Comment:
I don't think we need this new flag do we?  We include the root signature during container assembly unless StripRootSignature is provided.

Reply:
Ya you are right, the same thing can be accomplished checking the inputs.pRootSigBlob